fortran/fortran-rpn-calc/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+
+github.setup        scivision rpn-calc-fortran ec22a9780034188a691c4ce31b1778e5acf909ff
+version             1.4.1
+name                fortran-rpn-calc
+revision            0
+categories          fortran math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         RPN Reverse Polish Notation Calculator, in Fortran
+long_description    {*}${description}
+homepage            http://scivision.github.io/rpn-calc-fortran
```

Comment:
All links should be `https://` where possible

Reply:
Didn’t notice, sorry. Will fix now.